packages/tools/src/tools/base/BaseTool.ts

```diff
@@ -194,7 +194,7 @@ abstract class BaseTool implements IBaseTool {
   protected getTargetId(viewport: Types.IViewport): string | undefined {
     if (viewport instanceof StackViewport) {
       return `imageId:${viewport.getCurrentImageId()}`;
-    } else if (viewport instanceof VolumeViewport) {
+    } else if (viewport instanceof BaseVolumeViewport) {
```

Comment:
I event think this should be VolumeViewport too, since we don't have annotations yet for 3D viewport

Reply:
Will fix